### PR TITLE
GOV12.1.2 bootstrapping edit proposal

### PR DESCRIPTION
**Pull request by blimpa**
Originating forum post: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal/23879

### CODE REVIEW

File: MIP104/MIP104.md [hunk 1]
```diff
@@ -295,31 +297,41 @@ As long as Spark Protocol remains near its maximum borrow limit, the Stability F
 
 When NewGovToken and SubDAO farming comes into effect, DSR utilization will count both DSR use and NewGovToken and SubDAO farming use.
 
-##### 3.2.2.5: EDSR Upper Limit
+3.2.2.5: EDSR Upper Limit
```
Comment (VoteWizard) [inline MIP104/MIP104.md]:
```suggestion
#### 3.2.2.5: EDSR Upper Limit
```

File: MIP104/MIP104.md [hunk 2]
```diff
@@ -295,31 +297,41 @@ As long as Spark Protocol remains near its maximum borrow limit, the Stability F
...
 
 The EDSR Upper Limit acts as an effective limit to the EDSR, meaning that it can not exceed it even if the formula outputs a number above the limit. The EDSR Upper Limit can be altered by the Stability Facilitators via the weekly governance cycle.
 
-###### 3.2.2.5.1A
+3.2.2.5.1A
```
Comment (VoteWizard) [inline MIP104/MIP104.md]:
```suggestion
##### 3.2.2.5.1A
```

File: MIP104/MIP104.md [hunk 3]
```diff
@@ -295,31 +297,41 @@ As long as Spark Protocol remains near its maximum borrow limit, the Stability F
...
 
 ¤¤¤
 
-##### 3.2.2.6: EDSR Utilization-Based Multipliers
+#### 3.2.2.6: EDSR Utilization-Based Multipliers
 
 Each of the following EDSR multiplier tiers can be triggered by the Stability Facilitators based on the utilization rate. When first adopted, the EDSR begins at tier 1. If at any point in time the DSR utilization stays above the level needed for a different tier for a continuous period of more than 24 hours, the Stability Facilitator can choose to trigger the new tier, and a manual DSR adjustment must then be included in the next executive vote to account for the new tier.
 
-###### 3.2.2.6.1: EDSR tier 1
+3.2.2.6.1A: EDSR tier 1
```
Comment (VoteWizard) [inline MIP104/MIP104.md]:
```suggestion
###### 3.2.2.6.1A: EDSR tier 1
```

File: MIP104/MIP104.md [hunk 4]
```diff
@@ -295,31 +297,41 @@ As long as Spark Protocol remains near its maximum borrow limit, the Stability F
...
+
+¤¤¤
+
+Multiplier: 5x DSR
+
+Utilization: 0-35%
+
+¤¤¤
+
+3.2.2.6.2A: EDSR tier 2
```
Comment (VoteWizard) [inline MIP104/MIP104.md]:
```suggestion
###### 3.2.2.6.2A: EDSR tier 2
```

**Comment by 0xldr** [comment]
Poll passed. Good to merge - https://vote.makerdao.com/polling/QmQAtb17

**Review by 0xldr — APPROVED** [review]

**Pull request merged**